src/client/processor/npc/DueyProcessor.java

```diff
@@ -223,32 +224,97 @@ public static void deleteItem(MapleClient c, int packageId){
         c.announce(MaplePacketCreator.removeItemFromDuey(true, packageId));
     }
     
-     private static void addMesoToDB(int mesos, String sName, int recipientID) {
-        addItemToDB(null, 1, mesos, sName, recipientID);
+    // \/\/\/\/\/\/\/ METHODS FOR API \/\/\/\/\/\/\/\/
+    
+    // Sends a non-equipment item to an account
+    public static void sendItem(int itemId, int quantity, int mesos, String sender, int recvAcctId){
+        sendItem(itemId, quantity, mesos, -1, sender, 0, recvAcctId);
+    }
+    
+    // Sends a time-limited item to an account
+    public static void sendItem(int itemId, int quantity, int mesos, long timeLimit, String sender, int recvAcctId){
+        sendItem(itemId, quantity, mesos, timeLimit, sender, 0, recvAcctId);
+    }
+    
+    // Sends a time-limited item
+    public static void sendItem(int itemId, int quantity, int mesos, long timeLimit, String sender, int recipientId, int recvAcctId){
+        if (itemId < 2000000){
+            sendEquip(itemId, mesos, timeLimit, sender, recipientId, recvAcctId);
+        }
+        Item item = new Item(itemId, (short) 0, (short) quantity);
+        addItemToDB(item, quantity, mesos, sender, recipientId, recvAcctId, timeLimit);
```

Comment:
why does this method call `addItemToDB` twice if itemId < 20000?

Reply:
Fixed, thanks for the catch. ItemId < 2000000 indicates the item as an equipment.